test/data/brave_ads/single_page_application.html

```diff
@@ -0,0 +1,48 @@
+<html>
```

Comment:
How about we also add a multi-page application tests as a follow-up? If so, please raise an issue.

Reply:
https://github.com/brave/brave-browser/issues/39209